packages/render-fidelity-tools/src/artifact-creator.ts

```diff
@@ -165,7 +166,7 @@ export class ArtifactCreator {
   async captureScreenshot(
       renderer: string, scenarioName: string, dimensions: Dimensions,
       outputPath: string = join(this.outputDirectory, 'model-viewer.png')) {
-    const devicePixelRatio = 2;
+    const devicePixelRatio = 2;  // hard code?
```

Comment:
This is the DPR of our macbooks. It doesn't really make any difference, except it exercises our DPR codepath, so helps to catch more bugs.